src/smk/viewer-leaflet/layer/layer-vector-leaflet.js

```diff
@@ -305,6 +305,19 @@ include.module( 'layer-leaflet.layer-vector-leaflet-js', [ 'layer.layer-vector-j
             }
     }
 
+    function matches(value, condition) {
+        switch(condition.operator) {
+            case '>': return !Number.isNaN(value) && value > condition.value;
```

Comment:
You might want to consider throwing an error or somehow handle the situation where the value can't be parsed as a number. In the current state, these numerical comparisons would always return false and someone might be left scratching their heads wondering what's going wrong. Even a console.warn() message could be helpful.

Reply:
Thanks for this suggestion. I'll log a message when the value in a numerical comparison is not a number.